recipes-qt/automotive/neptune-ui/neptune.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Neptune
+After=dbus.service dbus-session.service systemd-user-sessions.service
+
+[Service]
+ExecStart=/usr/bin/appman -r -c /opt/am/config.yaml -c am-config.yaml --dbus none Main.qml -platform eglfs
```

Comment:
NeptuneUI should have dbus enabled by default: 

`ExecStart=/usr/bin/appman -r -c /opt/am/config.yaml --dbus session -c am-config.yaml Main.qml`